.gitignore

```diff
@@ -1 +1,7 @@
-dist
+/dist
```

Comment:
not to be super picky about this, but like the rest of my projects, node-and-npm-and-yarn related ignores should probably stay in `~/.gitignore` rather than project-specific files, because otherwise you add a new line for every editor-created tmp file that appears.

Reply:
I nearly committed node_modules, i didn't realise ppl used global gitignore files

Reply:
Yep, they save a lot of boilerplate over time - project-specific stuff like `dist` directories stick around in projects, but they avoid the case of someone using IntelliJ and adding .idea-project and then using vim and adding .swp, and so on and so forth. Same for npm stuff, like npm-error.log - that all goes in the global ignore.